gradle.properties

```diff
@@ -2,7 +2,7 @@ GROUP=se.ansman.kotshi
 VERSION_NAME=0.3.0-SNAPSHOT
```

Comment:
Is it possible to overwrite this in the upload script? I would like to append the current time.

Reply:
That's usually done automatically on the receiving side

Reply:
Oh, even better.